integration-cli/docker_cli_update_unix_test.go

```diff
@@ -22,7 +22,7 @@ func (s *DockerSuite) TestUpdateRunningContainer(c *check.C) {
 
 	memory := inspectField(c, name, "HostConfig.Memory")
 	if memory != "524288000" {
-		c.Fatalf("Got the wrong memory value, we got %d, expected 524288000(500M).", memory)
+		c.Fatalf("Got the wrong memory value, we got %s, expected 524288000(500M).", memory)
```

Comment:
Just a mention.
Can we use `c.Assert(memory, checker.Equals, "524288000")`? Then we don't need to care about `%s` any more.

By the way, I made some similar changes in another PR:  #19116
https://github.com/docker/docker/pull/19116/files#diff-221c6fb9a0aa122469c32da1609729deR24

I just happen to notice this problem when developing my own integration test case, but It's OK to change in this commit because in my PR they are totally unrelated. :smile: